P/pprof/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder
+
+name = "pprof"
+
+# PProf doesn't have proper versions
```

Comment:
:expressionless:

Reply:
Do we support pseudo versions?

https://golang.org/cmd/go/#hdr-Pseudo_versions

Reply:
I think VersionNumber accepts that, but BinaryBuilder.jl appends `+n`, where n is the build number, not sure they would play nicely together

Reply:
Oh, right...

We will see what @staticfloat says